### ISSUE TITLE
`PARAMETER` sole attribute not supported

### ISSUE THREAD

**Issue by Pranavchiku** [lfortran/lfortran#668]
```
semantic error: Attribute declaration not supported
  --> /home/pranavchiku/scipy/scipy/optimize/minpack2/dcstep.f:99:7
   |
99 |       parameter (zero=0.0d0,p66=0.66d0,two=2.0d0,three=3.0d0)
   |       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 


Note: if any of the above error or warning messages are not clear or are lacking
context please report it to us (we consider that a bug that needs to be fixed).
tokenizer error: Expecting terminating symbol for subroutine
```

**Comment by Pranavchiku** [comment]
the error originates from ```https://github.com/lfortran/lfortran/blob/main/src/lfortran/semantics/ast_common_visitor.h#L906```
with ```sym=zero```

**Comment by Pranavchiku** [comment]
resolved in #669

**Comment by Smit-create** [comment]
Fixed in #675
